tests/test_exonum_client.py

```diff
@@ -18,6 +18,16 @@
 
 SYSTEM_ENDPOINT_POSTFIX = "api/system/v1/{}"
 SERVICE_ENDPOINT_POSTFIX = "api/services/{}/{}"
+EXPLORER_ENDPOINT_POSTFIX = "api/explorer/v1/{}"
+
+
+def random_alphanumeric_string(length=32):
+    from uuid import uuid4
+
+    string = str(uuid4())
```

Comment:
I'd suggest to use `random` module with fixed seed, just so the test data will remain the same for every test.

Reply:
Should we really fix the seed?

Reply:
Yes, I think it's good. Later we can add fuzz tests with true random, but for unit tests the predictability is a good thing.